### ISSUE TITLE
visualbell

### ISSUE THREAD

**Issue by rmantel23** [VsVim/VsVim#2710]
When setting visualbell as expected the beep goes away, but there is no visual indication either. The behavior is as if ":set vb t_vb=" was given, but setting t_vb to something else doesn't seem to work. Is there a way to make the screen flash like in gVim?

**Comment by ricksladkey** [comment]
Yes, I too would like this feature and have been meaning to add it. I will take a look.
